config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main-upgrades.yaml

```diff
@@ -0,0 +1,162 @@
+periodics:
+- name: periodic-cluster-api-e2e-workload-upgrade-1-17-1-18-main
+  interval: 24h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
+    path_alias: sigs.k8s.io/cluster-api
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210201-cc9840e-1.19
+      args:
+        - runner.sh
+        - "./scripts/ci-e2e.sh"
+      env:
+        - name: KUBERNETES_VERSION_UPGRADE_FROM
```

Comment:
this stuff seems like it might be a pain to maintain in the future, any way we can put these default values in CAPI instead so we don't have to open a PR again test-infra every time we want to do a k8s version bump?

Reply:
This could be replaced by `stable-1.17` and so on (it is already supported by ci-e2e.sh)

I leave to @srm09 the call if to do this now, or after seeing the jobs working one time without adding dynamic version resolution on top.